### PR TITLE
docs: move to github pages

### PR DESCRIPTION
**Pull request by RaddishIoW**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
+[![GitHub issues](https://img.shields.io/github/issues/HaylLtd/cordova-background-geolocation-plugin?style=flat-square)](https://github.com/HaylLtd/cordova-background-geolocation-plugin/issues)
+[![GitHub stars](https://img.shields.io/github/stars/HaylLtd/cordova-background-geolocation-plugin?style=flat-square)](https://github.com/HaylLtd/cordova-background-geolocation-plugin/stargazers)
+![GitHub last commit](https://img.shields.io/github/last-commit/HaylLtd/cordova-background-geolocation-plugin?style=flat-square)
+
+![cordova](https://img.shields.io/badge/cordova-ios%20%7C%20android-blue?style=flat-square)
+
 ## Introduction
 
-Cross-platform geolocation for Cordova / PhoneGap with battery-saving "circular region monitoring" and "stop detection".
+### Cross-platform geolocation for Cordova / PhoneGap with battery-saving "circular region monitoring" and "stop detection"
```
Comment (HarelM) [inline README.md]: Remove PhoneGap?

File: README.md [hunk 2]
```diff
@@ -64,6 +50,26 @@ cordova plugin add cordova-background-geolocation-plugin \
...
+
+### Android background service issues
+
+There are repeatedly reported issues with some android devices not working in the background. Check if your device model is on  [dontkillmyapp list](https://dontkillmyapp.com) before you report new issue. For more information check out [dontkillmyapp.com](https://dontkillmyapp.com/problem).
+
+Another confusing fact about Android services is concept of foreground services. Foreground service in context of Android OS is different thing than background geolocation service of this plugin (they're related thought). **Plugin's background geolocation service** actually **becomes foreground service** when app is in the background. Confusing, right? :D
+
+If service wants to continue to run in the background, it must "promote" itself to `foreground service`. Foreground services must have visible notification, which is the reason why you can't disable drawer notification.
+
+The notification can only be disabled, when app is running in the foreground, by setting config option `startForeground: false` (this is the default option), but will always be visible in the background (if service was started).
```
Comment (HarelM) [inline README.md]: Cool! good work!

**Review by HarelM — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RaddishIoW) [reply README.md]: Done

**Review by HarelM — APPROVED** [review]
Amazing work!!

**Pull request merged**